macOSUpgrade.sh

```diff
@@ -192,7 +255,11 @@ verifyChecksum() {
 }
 
 cleanExit() {
-    /bin/kill "${caffeinatePID}"
+    /bin/ps -p "$caffeinatePID" > /dev/null && /bin/kill "$caffeinatePID"; wait "$caffeinatePID" 2>/dev/null
```

Comment:
You don't need to use `;` to put your script in one line, as we want to focus on readability.

Reply:
Replaced all kill process lines with a function, as it is used several times in the script.
View [3c0067c](https://github.com/kc9wwh/macOSUpgrade/pull/115/commits/3c0067c1bbf88e2ab4059688ad4c12a183d5dc85)